### ISSUE TITLE
something strange with CONSTANT=%i(https ssh)

### ISSUE THREAD

**Issue by borisaka** [lsegal/yard#1030]
My code:

``` ruby
class GithubRepository
      GITHUB_TRANSPORTS = %i(https ssh).freeze

      attr_writer :repo, :token, :host
      # Error wrong transport protocol
      class WrongGithubTransport < StandardError
        def message
          "only #{GITHUB_TRANSPORTS} allowed for now"
        end
      end
```

And then I see

```
GITHUB_TRANSPORTS =
        https ssh).freeze
```

![Screenshot](http://i.imgur.com/Gwnbeyj.png)

Also when I remove .freeze after symbols array – I see

```
GITHUB_TRANSPORTS =
        https ssh
```

(without brackets and %i)
OS X 10.11.6
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
yard 0.9.5

**Comment by lsegal** [comment]
I believe this is related to #1013

**Comment by lsegal** [comment]
Fixed by #1013
